pom.xml

```
@@ -16,7 +16,7 @@
         <java.version>17</java.version>
         <spring-cloud-azure.version>4.0.0</spring-cloud-azure.version>
         <jacoco.version>0.8.8</jacoco.version>
-        <pagopa-ecommerce-commons.version>1.9.6</pagopa-ecommerce-commons.version>
+        <pagopa-ecommerce-commons.version>1.10.0</pagopa-ecommerce-commons.version>
```

```suggestion
        <pagopa-ecommerce-commons.version>1.10.1</pagopa-ecommerce-commons.version>
```
To be updated after dependent commons PR merge